src/sentry/integrations/vsts/integration.py

```diff
@@ -422,6 +422,11 @@ def post_install(
             )
 
     def get_scopes(self) -> Sequence[str]:
+        if features.has(
+            "organizations:migrate-azure-devops-integration", self.pipeline.organization.id
+        ):
+            # This is the new way we need to pass scopes to the OAuth flow
+            return ("offline_access", "499b84ac-1321-427f-aa17-267ca6975798/.default")
```

Comment:
is the second part of the tuple okay to put in a public repo? link to docs?

Reply:
yeah it is, ill link docs here. tbh i want to put a link to the notion doc cause 1 link won't suffice this

Reply:
I just read the stackoverflow post. This is wild and I want to know who approved that PR on microsoft side! Anyways, @cathteng still not sure if I understand this fully but here's the docs on Microsoft side. It's ok to be public:
https://learn.microsoft.com/en-us/entra/identity-platform/v2-oauth2-auth-code-flow#request-an-authorization-code
https://learn.microsoft.com/en-us/rest/api/azure/devops/tokens/?view=azure-devops-rest-7.1&tabs=powershell#personal-access-tokens-pats